CHANGELOG.rst

```diff
@@ -1,3 +1,13 @@
+**JDBC Driver 3.13.12**
+
+- \| SNOW-473749 | Fix bug where time(3) objects weren't honoring USE_SESSION_TIMEZONE when called with getTimestamp() in Arrow format 
+- \| SNOW-499170 | Bringing GH actions up-to-date with Python connectors
+- \| SNOW-500624 | Fix JVM http proxy properties to work with PUT/GET 
+- \| SNOW-513708 | Refactor bind uploader
+- \| | BC FIPS update to latest 
```

Comment:
Mention this is Bouncy Castle instead of BC  and version number to which it is updated